framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -180,36 +185,59 @@ export const CreatableSelectDropdown = <T extends string = string> ({
   ]
 
   return (
-    <Box ref={ ref }>
-      <CreatableSelect
-        menuPlacement={ menuPlacement }
-        chakraStyles={ {
-          ...customSelectStyles,
-          container: (provided) => ({
-            ...provided,
-            width,
-          }),
-          option: (provided, { isSelected }) => ({
-            ...provided,
-            ...(isSelected && {
-              color: 'black',
+    <Box ref={ ref } w="sm" maxW="full">
+      { createNewOption ? (
+        <EditableText
+          value={ addFieldInputValue }
+          onChange={ (v) => { setAddFieldInputValue(v) } }
+          startWithEditView={ true }
+          submitOnBlur={ false }
+          leftItem={ undefined }
```

Comment:
not needed

Reply:
submitOnBlur and startWithEdtiView is needed

Reply:
I meant `leftItem`